values.yaml

```diff
@@ -1,83 +1,91 @@
+# values.yaml
+# For values.yaml configuration options and examples, refer to the values-example.yaml file.
+
 global:
+  # -- Set additional global labels.
   labels: {}
+  # -- Set additional global annotations.
   annotations: {}
-  fullNameOverride: "-"
-
-apps:
-  vaultwarden:
-    enabled: true
-    type: deployment
-    replicas: 1
-    initContainers:
-      copy-test:
-        image:
-          repository: busybox
-          tag: latest
-        command: ['sh', '-c', 'echo "test" > /data/test.txt']
-        volumeMounts:
-        - mountPath: /data
-          name: vaultwarden
-    containers:
-      vaultwarden:
-        image:
-          repository: vaultwarden/server
-          tag: 1.27.0-alpine
-        volumeMounts:
-        - mountPath: /data
-          name: vaultwarden
-        ports:
-        - name: http
-          containerPort: 80
-        envFrom:
-        - configMapRef:
-            name: vaultwarden
-    volumes:
-    - name: vaultwarden
-      persistentVolumeClaim:
-        claimName: vaultwarden
-
-services:
-  vaultwarden:
-    enabled: true
-    appName: ["vaultwarden"]
-    type: ClusterIP
-    ports:
-      http:
-        enabled: true
-        port: 8080
-        protocol: HTTP
-        targetPort: 80
-
-ingresses:
-  vaultwarden:
-    enabled: true
-    serviceName: vaultwarden
-    hosts:
-      - host: vaultwarden.example.com
-        paths:
-          - path: /
-            pathType: Prefix
-            service:
-              port: 8080
-    tls:
-    - hosts:
-        - vaultwarden.example.com
-      secretName: vaultwarden-tls-secret
+  # -- Set the full object prefix, defaults to releasName-ChartName if not set. This value takes precedence over nameOverride.
+  # Set to "-" to disable object name prefixing.
+  fullNameOverride:
+  # -- Set an override for the ChartName, defaults to ChartName if not set.
+  nameOverride:
+  # -- When `true``, the feature to automatically re-deploy an App's pods when a ConfigMap or Secret changes is enabled.
+  appReload: true
 
-configmaps:
-  vaultwarden:
-    enabled: true
-    data:
-      SOME_VAR: "some-value"
+defaults:
+  image:
+    pullPolicy: IfNotPresent
+  strategy: RollingUpdate
+  probes:
+    livenessProbe:
+      initialDelaySeconds: 0
+      periodSeconds: 10
+      timeoutSeconds: 5
+      successThreshold: 1
+      failureThreshold: 5
+      terminationGracePeriodSeconds: 30
+    readinessProbe:
+      initialDelaySeconds: 0
+      periodSeconds: 10
+      timeoutSeconds: 1
+      successThreshold: 1
+      failureThreshold: 5
+    startupProbe: {}
 
-persistence:
-  vaultwarden:
-    enabled: true
-    type: persistentVolumeClaim
-    persistentVolumeClaim:
-      spec:
-        accessModes:
-          - ReadWriteOnce
-        resources:
-          requests:
-            storage: 100Mi
+troubleshoot:
+  support-bundles:
+    replicated:
+      enabled: false
```

Comment:
I think it's fine to enable our support bundle by default. We're going to advise they do it anyway.